src/main/java/com/visualmetronome/VisualMetronomeConfig.java

```diff
@@ -63,10 +63,32 @@ default int tickCount()
 		return 1;
 	}
 
+	@ConfigItem(
```

Comment:
Didn't add an option to turn this feature off too, might be a good idea since people might not need all 3 numbers